ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/TraceFilter.java

```diff
@@ -88,6 +93,19 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
     final long toBlock = resolveBlockNumber(filterParameter.getToBlock());
     LOG.trace("Received RPC rpcName={} fromBlock={} toBlock={}", getName(), fromBlock, toBlock);
 
+    try {
+      if (maxRange > 0 && toBlock - fromBlock > maxRange)
+        throw new IllegalArgumentException(RpcErrorType.EXCEEDS_RPC_MAX_BLOCK_RANGE.getMessage());
+    } catch (IllegalArgumentException ex) {
+      LOG.atDebug()
+          .setMessage("eth_getLogs request {} failed:")
+          .addArgument(requestContext.getRequest())
+          .setCause(ex.getCause())
+          .log();
+      return new JsonRpcErrorResponse(
+          requestContext.getRequest().getId(), RpcErrorType.EXCEEDS_RPC_MAX_BLOCK_RANGE);
+    }
+
```

Comment:
we could remove the try-catch here and use a `if` instead

Reply:
Done